src/test/java/com/wikia/webdriver/elements/fandom/pages/HubPage.java

```diff
@@ -0,0 +1,28 @@
+package com.wikia.webdriver.elements.fandom.pages;
+
+import com.wikia.webdriver.elements.fandom.FandomPage;
+import com.wikia.webdriver.elements.fandom.components.HeroUnit;
+import lombok.Getter;
+
+/**
+ * Created by Ludwik on 2016-07-29.
+ */
+public class HubPage extends FandomPage<HubPage>{
+
+  @Override
+  public HubPage open() {
+    getUrl(String.format("http://qa.fandom.wikia.com/%s", "games"));
+
+    return this;
+  }
+
+    @Getter(lazy = true)
+    private final HeroUnit heroUnit = new HeroUnit();
+    private String URL = "http://qa.fandom.wikia.com/";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this field "URL" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00116)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1213)